src/plone/base/interfaces/__init__.py

```diff
@@ -26,7 +26,6 @@
 from .controlpanel import ITinyMCEPluginSchema
 from .controlpanel import ITinyMCEResourceTypesSchema
 from .controlpanel import ITinyMCESchema
-from .controlpanel import ITinyMCESpellCheckerSchema
```

Comment:
Question: Do we need an upgrade step and remove the records from the registry?

Reply:
yes 👍🏼

Reply:
@reinhardt is doing that in https://github.com/plone/plone.app.upgrade/pull/328
We are testing them all together now on Jenkins.